src/main/java/graphql/execution/instrumentation/Instrumentation.java

```diff
@@ -72,62 +71,26 @@ default InstrumentationState createState() {
     InstrumentationContext<List<ValidationError>> beginValidation(InstrumentationValidationParameters parameters);
 
     /**
-     * This is called just before the data fetching stage is started and finishes as soon as the query is dispatched ready for completion.  This
-     * is different to {@link #beginDataFetch(graphql.execution.instrumentation.parameters.InstrumentationDataFetchParameters)}
-     * in that this step does not wait for the values to be completed, only dispatched for completion.
-     *
-     * @param parameters the parameters to this step
-     *
-     * @return a non null {@link InstrumentationContext} object that will be called back when the step ends
-     */
-    default InstrumentationContext<CompletableFuture<ExecutionResult>> beginDataFetchDispatch(InstrumentationDataFetchParameters parameters) {
-        return (result, t) -> {
-        };
-    }
-
-    /**
-     * This is called just before the data fetching stage is started, waits for all data to be completed and when this step finishes the {@link InstrumentationContext#onEnd(Object, Throwable)}
-     * will be called indicating that the step has finished.
```

Comment:
beginDataFetchDispatch was a badly named step in retrospect.  It is called when the query operation is started hence the new name which is less confusing